frontend/src/pages/login/Login.tsx

```diff
@@ -1,6 +1,6 @@
-import { Dispatch, useState } from 'react';
+import React, { Dispatch, useState } from 'react';
 import { useHistory } from 'react-router-dom';
-import { TextField, Button } from '@material-ui/core';
+import { TextField, Button, Link } from '@material-ui/core';
```

Comment:
You used Material UI's link instead of React-Router's. Lol.
```suggestion
import { TextField, Button} from '@material-ui/core';
```

Reply:
Hahaha ok